WooCommerce/Classes/POS/Presentation/TotalsView.swift

```diff
@@ -387,10 +395,14 @@ private extension View {
 
 #if DEBUG
 #Preview {
-    let totalsVM = TotalsViewModel(orderService: POSOrderPreviewService(),
-                                   cardPresentPaymentService: CardPresentPaymentPreviewService(),
-                                   currencyFormatter: .init(currencySettings: .init()),
-                                    paymentState: .acceptingCard)
-    return TotalsView(viewModel: totalsVM)
+    let posModel = PointOfSaleAggregateModel(
+        itemProvider: POSItemProviderPreview(),
+        cardPresentPaymentService: CardPresentPaymentPreviewService(),
+        orderService: POSOrderPreviewService())
+    let totalsVM = TotalsViewModel(
+        posModel: posModel,
+        cardPresentPaymentService: CardPresentPaymentPreviewService(),
+        paymentState: .acceptingCard)
+    TotalsView(viewModel: totalsVM)
```

Comment:
I think I've got that issue in a bunch of places.

I really want previews to work, and I'm trying to figure out how we can modularise just the UI without causing ourselves future difficulties, because having this stuff in a separate module seems like the only way.

I'll go through and fix these for every view separately.